verification/verification_test.go

```diff
@@ -99,13 +99,14 @@ func GetTestTarget(support_needed []string) (TestDPEInstance, error) {
 // Get the emulator target
 func GetEmulatorTarget(support_needed []string) (TestDPEInstance, error) {
 
-	value := reflect.ValueOf(DpeEmulator{}.supports)
+	/*value := reflect.ValueOf(DpeEmulator{}.supports)
 	for i := 0; i < len(support_needed); i++ {
 		support := reflect.Indirect(value).FieldByName(support_needed[i])
 		if !support.Bool() {
 			return nil, errors.New("Error in creating dpe instances - supported feature is not enabled in emulator")
 		}
-	}
+	}*/
+	// TODO : Get the supported modes from emulator and then check.
```

Comment:
Would implementing this be a large enough effort to put in a separate PR? I think it might be okay to do as part of this PR.

Reply:
I might be misunderstanding how this works. But could you do the same as `GetSimulatorTarget` below? i.e.

```Rust
	value := reflect.ValueOf(DpeEmulator{}.supports)
	fields := reflect.Indirect(value)
	fVal := reflect.New(reflect.TypeOf(DpeEmulator{}.supports))
	for i := 0; i < len(support_needed); i++ {
		for j := 0; j < value.NumField(); j++ {
			if fields.Type().Field(j).Name == support_needed[i] {
				fVal.Elem().Field(j).SetBool(true)
			}
		}
	}
```

Reply:
In general I agree we should be able to use Zach's code to do this in this PR. This will also require calling GetProfile, but I think that can be done as long as you construct the emulator and then do this check.

Reply:
Updated the code, based on the comments provided.